### PR TITLE
Add template processing to Pdfexport.streamPdfFromHtml method, fixes …

### PR DESCRIPTION
**Pull request by bytemine-gmbh**
…https://github.com/Icinga/icingaweb2-module-reporting/issues/85 .

### CODE REVIEW

File: library/Pdfexport/ProvidedHook/Pdfexport.php [hunk 1]
```diff
@@ -66,7 +66,24 @@ public function htmlToPdf($html)
...
+                    ->addAttributes($html->getAttributes())
+                    ->removeMargins()
+                )
+                ->toPdf();
+
+            $merger = new Merger(new TcpdiDriver());
+            $merger->addRaw($coverPagePdf);
+            $merger->addRaw($pdf);
+
+	    $pdf = $merger->merge();
```
Comment (nilmerg) [inline library/Pdfexport/ProvidedHook/Pdfexport.php:84]: Please keep your indentation consistent (Add two missing spaces here)

File: library/Pdfexport/ProvidedHook/Pdfexport.php [hunk 2]
```diff
@@ -66,7 +66,24 @@ public function htmlToPdf($html)
...
+	}
+	return $pdf;
```
Comment (nilmerg) [inline library/Pdfexport/ProvidedHook/Pdfexport.php:86]: Please add an empty line after closing braces

**Review by nilmerg — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've left some comments, once these are solved, I'll merge this right away! :)

File: library/Pdfexport/ProvidedHook/Pdfexport.php (hunk 2 above)
Comment (dra) [reply library/Pdfexport/ProvidedHook/Pdfexport.php:87]: Thanks, I just changed the code according to your suggestions.

File: library/Pdfexport/ProvidedHook/Pdfexport.php [hunk 3]
```diff
@@ -66,7 +66,25 @@ public function htmlToPdf($html)
...
+                ->toPdf();
+
+            $merger = new Merger(new TcpdiDriver());
+            $merger->addRaw($coverPagePdf);
+            $merger->addRaw($pdf);
+
+            $pdf = $merger->merge();
+	}
+
+	return $pdf;
```
Comment (nilmerg) [inline library/Pdfexport/ProvidedHook/Pdfexport.php:87]: Oops, didn't saw that previously. The entire block is not indented correctly. We use 4 spaces everywhere.

Comment (dra) [reply library/Pdfexport/ProvidedHook/Pdfexport.php]: fixed indention to 4 spaces

**Comment by nilmerg** [comment]
Looks fine to me now, but please rebase with master now. I've setup stylechecks there now and so they run for your changes as well. Ignore the checks for PHP 5.6 and 7.0, but take adjustments according to the other results. Thanks.

**Comment by dra** [comment]
> Looks fine to me now, but please rebase with master now. I've setup stylechecks there now and so they run for your changes as well. Ignore the checks for PHP 5.6 and 7.0, but take adjustments according to the other results. Thanks.

Hi, just rebased and pushed again.

**Review by nilmerg — APPROVED** [review]
LGTM!

**Pull request merged**